src/testdir/test_cmdline.vim

```diff
@@ -3412,7 +3412,7 @@ func Test_fuzzy_completion_mapname()
   nmap <Plug>state :
   nmap <Plug>FendingOff :
   call feedkeys(":nmap <Plug>fo\<C-A>\<C-B>\"\<CR>", 'tx')
-  call assert_equal("\"nmap <Plug>format <Plug>TestFOrmat <Plug>FendingOff <Plug>goformat <Plug>fendoff", @:)
+  call assert_equal("\"nmap <Plug>format <Plug>FendingOff <Plug>fendoff <Plug>TestFOrmat <Plug>goformat", @:)
```

Comment:
@habamax Could you please test it when you are available thank you . and the order here has changed. because the current algorithm focuses more on continuous matching...hope it's fine.